packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.component.tsx

```diff
@@ -25,7 +25,8 @@ export const DobField: React.FC = () => {
   const { t } = useTranslation();
   const {
     fieldConfigurations: { dateOfBirth },
-  } = useConfig() as RegistrationConfig;
+  } = useConfig<RegistrationConfig>();
+  const allowEstimatedBirthDate = dateOfBirth?.allowEstimatedDateOfBirth;
   const [dobUnknown] = useField('birthdateEstimated');
   const dobKnown = !dobUnknown.value;
```

Comment:
It's a little weird to create a variable which is `!dobUnknown.value` and then use it as `!dobKnown`. It might read more clearly just to have `dobUnknown.value` in the OR, no?

Reply:
Can this be addressed?